tests/Behat/Context/Transform/OrderContext.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tests\Sylius\RefundPlugin\Behat\Context\Transform;
+
+use Behat\Behat\Context\Context;
+use Sylius\Component\Core\Model\OrderInterface;
+use Sylius\Component\Core\Repository\OrderRepositoryInterface;
+use Webmozart\Assert\Assert;
+
+final class OrderContext implements Context
```

Comment:
Couldn't you use a transformer from Sylius?

Reply:
no, because in the core we don't have a transformer from a regular expression.

Reply:
And is it necessary? Couldn't you write the step without regular expression (if it is possible, it would be more transparent)?